doc/meson.build

```diff
@@ -0,0 +1,63 @@
+xsltproc_args = [
+  '--nonet',
+  '--stringparam', 'man.output.quietly', '1',
+  '--stringparam', 'funcsynopsis.style', 'ansi',
+  '--stringparam', 'man.th.extra1.suppress', '1',
+  '--stringparam', 'man.authors.section.enabled', '0',
+  '--stringparam', 'man.copyright.section.enabled', '0',
+]
+xsltproc = find_program('xsltproc')
+xmlto = find_program('xmlto')
+
+# TODO: is the docbook DTD/XSL catalog check needed or does xsltproc take care of that?
+configured_xml = configure_file(
+  configuration: {
+    'VERSION': meson.project_version(),
+    # FIXME: that's not a good way of this it…
+    'srcdir': meson.current_source_dir(),
+  },
+  input: 'flatpak-builder-docs.xml.in',
+  # depend_files:
+  output: 'flatpak-builder-docs.xml',
+)
+
+man_gen_command = [
+  xsltproc, xsltproc_args,
+  'http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl',
+]
+
+man_pages = [
+  ['flatpak-builder', '1'],
+  ['flatpak-manifest', '5'],
+]
+foreach man_page : man_pages
+  man_page_name = '@0@.@1@'.format(man_page[0], man_page[1])
+  custom_target(man_page[0] + '-man',
+    command: [man_gen_command, man_page_name],
+    depend_files: man_page[0] + '.xml',
+    output: man_page_name,
+    install: false,
+  )
+  install_man(man_page_name)
```

Comment:
Unfortunately, this will not actually work since install_man expects to find files in the source directory at configure time. You can just set the custom target to install: true and `install_dir: join_paths(get_option('mandir'), 'man'+man_page[1])`